src/core/grpc_service_v2.proto

```diff
@@ -1123,7 +1123,7 @@ message CudaSharedMemoryRegisterRequest
 
   //@@  .. cpp:var:: bytes raw_handle
   //@@
-  //@@     The raw serialized cudaIPC handle.
+  //@@     The raw serialized cudaIPC handle in base64 encoding.
```

Comment:
For protobuf we don't need to base64 endode this. For your python layer you can use base64... to make easier to visualize/debug the handle. But for here you should just encode the raw bytes, which means you will have to base64 decode in the library when constructing the message.

Reply:
Done